README.md

```diff
@@ -67,41 +71,15 @@ If you want to register specific Python versions, then use
 `python.toolchain()` for each version you need:
 
 ```starlark
+# Update the version "0.0.0" to the a release found here: 
+# https://github.com/bazelbuild/rules_python/releases.
+bazel_dep(name = "rules_python", version = "0.0.0")
 python = use_extension("@rules_python//python:extensions.bzl", "python")
 
 python.toolchain(
-    python_version = "3.9",
-)
-```
-
-### Using pip with bzlmod
-
-NOTE: bzlmod support is still experimental; APIs subject to change.
-
-To use dependencies from PyPI, the `pip.parse()` extension is used to
-convert a requirements file into Bazel dependencies.
-
-```starlark
-python = use_extension("@rules_python//python/extensions:python.bzl", "python")
-python.toolchain(
-    python_version = "3.9",
-)
-
-interpreter = use_extension("@rules_python//python/extensions:interpreter.bzl", "interpreter")
-interpreter.install(
-    name = "interpreter",
-    python_name = "python_3_9",
+    python_version = "3.11",
 )
-use_repo(interpreter, "interpreter")
-
-pip = use_extension("@rules_python//python/extensions:pip.bzl", "pip")
-pip.parse(
-    hub_name = "pip",
-    python_interpreter_target = "@interpreter//:python",
-    requirements_lock = "//:requirements_lock.txt",
-    requirements_windows = "//:requirements_windows.txt",
-)
-use_repo(pip, "pip")
+use_repo(python, "python_3_11", "python_aliases")
```

Comment:
This use_repo() needs more explanation or usage examples, otherwise omit it. People are going to copy/paste this without understanding why they do (or don't) need it.